### ISSUE TITLE
buildSearchableOptions error message

### ISSUE THREAD

**Issue by ice1000** [JetBrains/gradle-intellij-plugin#470]
I saw

```
2020-03-27 05:15:02,992 [   1723]   WARN - j.internal.DebugAttachDetector - Unable to start DebugAttachDetector, please add `--add-exports java.base/jdk.internal.vm=ALL-UNNAMED` to VM options
```

during building searchable options.
Also, I got this exception

```
2020-03-27 05:15:16,963 [  15694]  ERROR - tellij.openapi.util.ObjectTree - Already shutdown
java.util.concurrent.RejectedExecutionException: Already shutdown
        at com.intellij.util.concurrency.BoundedTaskExecutor.execute(BoundedTaskExecutor.java:133)
        at java.base/java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:118)
        at com.intellij.openapi.vfs.impl.local.FileWatcher.setWatchRoots(FileWatcher.java:165)
        at com.intellij.openapi.vfs.impl.local.WatchRootsManager.updateFileWatcher(WatchRootsManager.java:151)
        at com.intellij.openapi.vfs.impl.local.WatchRootsManager.replaceWatchedRoots(WatchRootsManager.java:75)
        at com.intellij.openapi.vfs.impl.local.LocalFileSystemImpl.replaceWatchedRoots(LocalFileSystemImpl.java:135)
        at com.intellij.openapi.vfs.LocalFileSystem.removeWatchedRoots(LocalFileSystem.java:91)
        at com.intellij.compiler.CompilerManagerImpl.lambda$new$0(CompilerManagerImpl.java:103)
        at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:111)
        at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
        at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
        at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:95)
        at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
        at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
        at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:134)
        at com.intellij.openapi.util.Disposer.dispose(Disposer.java:116)
        at com.intellij.openapi.util.Disposer.dispose(Disposer.java:106)
        at com.intellij.openapi.project.impl.ProjectManagerImpl.dispose(ProjectManagerImpl.java:168)
        at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:111)
        at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
        at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
        at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:95)
        at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
        at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
        at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:134)
        at com.intellij.openapi.util.Disposer.dispose(Disposer.java:116)
        at com.intellij.openapi.util.Disposer.dispose(Disposer.java:106)
        at com.intellij.serviceContainer.ComponentManagerImpl.dispose(ComponentManagerImpl.kt:846)
        at com.intellij.openapi.application.impl.ApplicationImpl.dispose(ApplicationImpl.java:350)
        at com.intellij.openapi.util.ObjectNode.lambda$execute$0(ObjectNode.java:111)
        at com.intellij.openapi.util.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:184)
        at com.intellij.openapi.util.ObjectNode.execute(ObjectNode.java:71)
        at com.intellij.openapi.util.ObjectTree.executeAll(ObjectTree.java:134)
        at com.intellij.openapi.util.Disposer.dispose(Disposer.java:116)
        at com.intellij.openapi.util.Disposer.dispose(Disposer.java:106)
        at com.intellij.openapi.application.impl.ApplicationImpl.lambda$disposeContainer$3(ApplicationImpl.java:181)
        at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:976)
        at com.intellij.openapi.application.impl.ApplicationImpl.disposeContainer(ApplicationImpl.java:179)
        at com.intellij.openapi.application.impl.ApplicationImpl.disposeSelf(ApplicationImpl.java:200)
        at com.intellij.openapi.application.impl.ApplicationImpl.doExit(ApplicationImpl.java:621)
        at com.intellij.openapi.application.impl.ApplicationImpl.exit(ApplicationImpl.java:589)
        at com.intellij.openapi.application.impl.ApplicationImpl.exit(ApplicationImpl.java:578)
        at com.intellij.ide.ui.search.TraverseUIStarter.main(TraverseUIStarter.java:81)
        at com.intellij.openapi.application.ApplicationStarter.main(ApplicationStarter.java:62)
        at com.intellij.idea.ApplicationLoader$startApp$8$1$1.run(ApplicationLoader.kt:228)
        at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:94)
        at com.intellij.idea.ApplicationLoader$startApp$8$1.run(ApplicationLoader.kt:227)
        at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:201)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java
:831)
        at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeLater$4(ApplicationImpl.java:310)
        at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:80)
        at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:128)
        at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
        at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:184)
        at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
        at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
        at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85
)
        at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
        at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:974)
        at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:847)
        at com.intellij.ide.IdeEventQueue.lambda$null$8(IdeEventQueue.java:449)
        at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:739)
        at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
        at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java
:831)
        at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
        at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
        at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
        at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
        at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
2020-03-27 05:15:16,965 [  15696]  ERROR - tellij.openapi.util.ObjectTree - IntelliJ IDEA 2020.1 Beta  Build #IC-201.6668.1
3
2020-03-27 05:15:16,965 [  15696]  ERROR - tellij.openapi.util.ObjectTree - JDK: 11.0.6; VM: OpenJDK 64-Bit Server VM; Vend
or: JetBrains s.r.o
2020-03-27 05:15:16,965 [  15696]  ERROR - tellij.openapi.util.ObjectTree - OS: Windows 10
```

**Comment by zolotov** [comment]
see https://youtrack.jetbrains.com/issue/IDEA-234557, should be fixed >201.6746 SDK

**Comment by ice1000** [comment]
Gotcha
